Remotion/ObjectBinding/Web/ObjectBinding.Web.csproj

```diff
@@ -18,6 +18,9 @@
       <PackagePath>res/HTML</PackagePath>
     </Content>
   </ItemGroup>
+  <ItemGroup>
+    <None Update="..\..\Remotion LGPL.licenseheader" Pack="true" PackagePath="res/HTML" />
```

Comment:
We also should apply `Visible=false`.
see https://re-motion.atlassian.net/browse/RMSRCBUILD-181 for reference.

Reply:
Since we only update the item (and not add it) we should evaluate if `Visible="False"` is really wanted here. The licenseheader is added in `Shared.Build.props` and now it is invisible only in this project.

If we want it invisible we should change it in `Shared.Build.props` otherwise we should remove the attribute again.